pkg/console/subresource/deployment/deployment.go

```diff
@@ -231,23 +234,37 @@ func GetLogLevelFlag(logLevel operatorv1.LogLevel) string {
 	return flag
 }
 
+func withLogLevelFlag(logLevel operatorv1.LogLevel, flags []string) []string {
+	if logLevelFlag := GetLogLevelFlag(logLevel); logLevelFlag != "" {
+		return append(flags, logLevelFlag)
+	}
+	return flags
+}
+
+func withStatusPageFlag(providers operatorv1.ConsoleProviders, flags []string) []string {
```

Comment:
Just a follow up, but I'm a bit surprised we need this to direct set the flag when we already have:

https://github.com/openshift/console/blob/c0b0efd8ac532b6af16e952cb7022efd2b6aab18/pkg/serverconfig/config.go#L123

Which should have read the config file & set the flag for us.  I think that's the direction we want all of our configs to be, really. 

Something we can look at yet.

Reply:
Also, it's weird that the console reads a yaml & then sets flags imho :)